src/reports/models/reportsManager.js

```diff
@@ -50,6 +50,20 @@ module.exports.editReport = async (testId, reportId, reportBody) => {
     await databaseConnector.updateReport(testId, reportId, { notes });
 };
 
+module.exports.deleteReport = async (testId, reportId) => {
+    let reportSummary = await databaseConnector.getReport(testId, reportId);
+    let config = await configHandler.getConfig();
+    let report = await getReportResponse(reportSummary[0], config);
+
+    if (!FINAL_REPORT_STATUSES_WITH_END_TIME.includes(report.status)) {
+        let error = new Error(`Can't delete running test with status ${report.status}`);
+        error.statusCode = 400;
```

Comment:
i'd return here 429 and add to swagger since it's a conflict in the state of the report rather than a bad request

Reply:
good point